packages/recommend-vdom/src/FacetsView.tsx

```diff
@@ -23,11 +23,11 @@ export function createFacetsView({ createElement, Fragment }: Renderer) {
               key={item.facetValue}
               className={cx('auc-Recommend-item', props.classNames.item)}
             >
-              <props.itemComponent
-                createElement={createElement}
-                Fragment={Fragment}
-                item={item}
-              />
+              {props.itemComponent({
```

Comment:
Using it as a function allow to return VNode[]. It is not possible with the <props.itemComponent /> syntax. 

(Error triggered: `  Its return type 'VNode<any> | VNode<any>[]' is not a valid JSX element.`)

Reply:
I think this can cause problems because it means `createElement()` won't get called, and the underlying framework won't be able to interact with this function. For instance, you won't be able to use React Context in there. That could be problematic if users use a `<Link>` from Next.js in their item component.

Not 100% sure but we should try that first.